static/css/components/page-banner.less

```diff
@@ -7,13 +7,19 @@
 
 .page-banner {
   display: block;
-  color: @white;
+  color: @black;
```

Comment:
```suggestion
  color: @white;
```
Black on a dark grey background is very hard to read.

Reply:
Good catch, this was only supposed to be on `-mybooks`